lib/linux.js

```diff
@@ -59,11 +80,25 @@ module.exports = async () => {
 		execFile(xwininfoBin, xpropDetailsArgs.concat([activeWindowId]))
 	]);
 
-	return parseLinux({
+	const data = parseLinux({
 		activeWindowId,
 		boundsStdout,
 		stdout
 	});
+	if (data.owner && typeof data.owner.processId === 'number') {
```

Comment:
In what situations would the owner and owner.processId not be available?

Reply:
`owner` will really always be set, ass it create unconditionally.
As for `processId`, for example if `_NET_WM_PID` is not set. I don't know X ecosystem so well to know existing implementations which work this way, technically one could write his own implementation not setting it I suppose. 
Also, once could just use `xprop -set` to set it to something else, and still stay in valid ecosystem, as this is an optional atom.

Reply:
Also, there is a chance the info will be incorrect, as it may be a valid pid, but from another computer, not the one we are running at.

Reply:
Removed `owner` check